src/osd/PrimaryLogPG.cc

```diff
@@ -15737,7 +15737,11 @@ int PrimaryLogPG::getattr_maybe_cache(
 	*val = i->second;
       return 0;
     } else {
-      return -ENODATA;
+      if (obc->obs.exists) {
```

Comment:
Maybe add the new NOENT possible return value to the
(nonexistent) getattr_maybe_cache() description in the .h file?

Reply:
@ronen-fr the return value for erasure code pools was different from rep. pools, it's not new return code that added, only matching one.

Reply:
Add a comment to getattr_maybe_cache:

```
/**
 * getattr_maybe_cache
 *
 * Populates val (if non-null) with the value of the attr with the specified key.
 * Returns -ENOENT if object does not exist, -ENODATA if the object exists,
 * but the specified key does not.
 */
```